### DISCUSSION TITLE
XIAO eInk Expansion Board | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1298] [Q&A]
# XIAO eInk Expansion Board | Seeed Studio Wiki

Getting Started XIAO eInk Expansion Board

https://wiki.seeedstudio.com/XIAO-eInk-Expansion-Board/

**Comment by big-kita** [comment]
Is this expansion board compatible with the XIAO ESP32-C6?
